dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3659,11 +3657,12 @@ LRESULT CShellBrowser::OnOrganizeFavorites(WORD wNotifyCode, WORD wID, HWND hWnd
 LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL &bHandled)
 {
     fStatusBarVisible = !fStatusBarVisible;
-    if (fStatusBar)
```

Comment:
Why is this check not wanted anymore?

Reply:
Because there is no possible condition where ```fStatusBar``` would not be initialized.

If it is enabled on launch, then ```fStatusBar``` will be initialized and you can show or hide it when toggling it.

If it is disabled on launch, then it will be created before calling the show window function.

Reply:
I still do not see the difference with existing code. Is the existing check useless (with regard to existing code), then?
If it is, then would it be useless at https://github.com/reactos/reactos/blob/0a3b5d04899a40fbfe77d31dc7cad9d42c5cbcfd/dll/win32/browseui/shellbrowser.cpp#L1444 too?

Reply:
> Because there is no possible condition where fStatusBar would not be initialized.

Unless, due to a low resources (i.e. memory) condition, the status bar window creation failed.

Reply:
Good point @HBelusca. I reverted this change.

Reply:
While here, would the 3 existing calls without an `fStatusBar` check need one?